ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/fabric/bgp_overlay/switch-route-maps-evpn.j2

```diff
@@ -33,10 +33,10 @@
         type: permit
         set:
 {%     if leaf.mlag == true and leaf.mlag_role == "primary" %}
-          - "extcommunity soo {{ vtep_loopback_network_summary | ipaddr('network') | ipmath(leaf.id + spine.nodes | length) }}:1 additive"
+          - "extcommunity soo {{ vtep_loopback_network_summary | ipaddr('network') | ipmath(leaf.id + switch.max_spines) }}:1 additive"
```

Comment:
Should not we protect using a var built in another logic file? If another designs leverage this file without computing `switch.max_spines` it won't work

Adding a header with variables required from `switch` namespace could help